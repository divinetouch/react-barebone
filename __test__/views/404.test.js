import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../app/views/404.js';


it('should produce a page not found div', () => {
    const component = renderer.create(<NotFound />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});