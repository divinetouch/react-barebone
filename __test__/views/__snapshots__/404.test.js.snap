exports[`test should produce a page not found div 1`] = `
<div>
  <h2
    className="ui center aligned header"
    style={
      Object {
        "color": "red",
        "fontWeight": "bold"
      }
    }>
    Page Not Found
  </h2>
</div>
`;
